Mathlib/RingTheory/AdicCompletion/Algebra.lean

```diff
@@ -215,6 +336,8 @@ instance : IsScalarTower R (AdicCompletion I R) (AdicCompletion I M) where
     ext n
     rw [smul_eval, val_smul, val_smul, smul_eval, smul_assoc]
 
+unseal smul' in
+unseal mul in
 /-- A priori `AdicCompletion I R` has two `AdicCompletion I R`-module instances.
 Both agree definitionally. -/
 example : module I = @Algebra.toModule (AdicCompletion I R)
```

Comment:
I think the two `unseal`s here are a problem: In the other PR in the `AsTensorProduct` file (https://github.com/leanprover-community/mathlib4/blob/abf5c5635c9bde7cefa76b2c18405067b604e089/Mathlib/RingTheory/AdicCompletion/AsTensorProduct.lean#L133-L138), I had to add the same `unseal`s because Lean can no longer identify these two instances as equal. Is there anything I can do about this? Or does it mean we can't make `mul` and `smul'` irreducible?